x/concentrated-liquidity/fees_test.go

```diff
@@ -699,3 +701,125 @@ func (s *KeeperTestSuite) TestCollectFees() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestUpdateFeeAccumulatorPosition() {
+	var (
+		ownerOne = s.TestAccs[0]
+		ownerTwo = s.TestAccs[1]
+	)
+
+	type updateFeeAccumPositionTest struct {
+		poolId        uint64
+		owner         sdk.AccAddress
+		liquidity     sdk.Dec
+		lowerTick     int64
+		upperTick     int64
+		expectedError error
+	}
+
+	positions := map[int]updateFeeAccumPositionTest{
+		1: {
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		2: {
+			lowerTick: DefaultLowerTick + 1,
+			upperTick: DefaultUpperTick,
+		},
+		3: {
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick + 1,
+		},
+	}
+
+	tests := map[string]updateFeeAccumPositionTest{
+		"update position with a different pool ID": {
+			poolId:    2,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different owner": {
+			poolId:    1,
+			owner:     ownerTwo,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different lower tick": {
+			poolId:    1,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick + 1,
+			upperTick: DefaultUpperTick,
+		},
+		"update position with a different upper tick": {
+			poolId:    1,
+			owner:     ownerOne,
+			liquidity: DefaultLiquidityAmt,
+			lowerTick: DefaultLowerTick,
+			upperTick: DefaultUpperTick + 1,
+		},
+		"err: pool does not exist": {
+			poolId:        3,
+			expectedError: cltypes.PoolNotFoundError{PoolId: 3},
+		},
+		"err: position does not exist": {
+			poolId:        1,
+			owner:         ownerOne,
+			liquidity:     DefaultLiquidityAmt,
+			lowerTick:     DefaultLowerTick - 1,
+			upperTick:     DefaultUpperTick,
+			expectedError: accum.NoPositionError{Name: fmt.Sprintf("%s,%s,%d,%d", "1", ownerOne.String(), DefaultLowerTick-1, DefaultUpperTick)},
```

Comment:
nit: there is a helper function to perform this formatting: https://github.com/osmosis-labs/osmosis/blob/b2e2d3c5d7fe94ea9c3c87fe074dbc05c666e9da/x/concentrated-liquidity/fees.go#L255